### ISSUE TITLE
Crash with serene seasons

### ISSUE THREAD

**Issue by iamsoNewBee** [GTNewHorizons/Angelica#717]
### Your GTNH Discord Username

null

### Mod Version

bate 16

### Bug Report

[crash-2024-11-02_10.45.21-server.txt](https://github.com/user-attachments/files/17605336/crash-2024-11-02_10.45.21-server.txt)

Crash when i enter a map

### Java Version

Java 8

### Graphics Card Vendor

Other (Please Specify)

### Shaders

No

### MCPF or other features disabled by default

No: Go ahead and create an issue

### Mod List or GTNH Pack Version

Basically serene season and angelica.

### Final Checklist

- [X] I have searched the issues and haven't found a similar issue.
- [X] I have read the known incompatibilities and this is not related to one of those.
- [X] I am running an officially released version.  (Or, if I've compiled it myself I plan to fix the issue)

**Comment by iamsoNewBee** [comment]
If its possible to be fixed?

**Comment by DarkShadow44** [comment]
Seems to work fine for me, having compiled Angelica and SereneSeasons from latest source. Is this still an issue, and if so, how can I reproduce it?

**Comment by iamsoNewBee** [comment]
When I try to enter the map, the game crash.
[crash-2025-02-18_02.32.14-server.txt](https://github.com/user-attachments/files/18830777/crash-2025-02-18_02.32.14-server.txt)
![Image](https://github.com/user-attachments/assets/ca3e626c-29c1-4d0a-80cf-4aadab51ba0a)

**Comment by DarkShadow44** [comment]
Recompile SereneSeasons from source, you're way out of date. I'd provide binaries, but I'm legally unable to do so.

**Comment by iamsoNewBee** [comment]
Out of date? I will have a check. I download it form cf.

**Comment by DarkShadow44** [comment]
> Out of date? I will have a check. I download it form cf.

Yeah you can't do that. Need to build it yourself.

**Comment by iamsoNewBee** [comment]
Any other ways to get it?

**Comment by DarkShadow44** [comment]
not legally, no

**Comment by iamsoNewBee** [comment]
U mean it is an unreleased version?

**Comment by DarkShadow44** [comment]
Yes, I got the permission to backport it, but I can't share releases. Please don't bother the original authors about that though, they already regret giving me permission to do the backport

**Comment by Cleptomania** [comment]
Gonna close this because I don't even think this is an Angelica compat issue? The provided crash log is a server crash.

**Comment by DarkShadow44** [comment]
Yeah, if the issue should happen with latest git (which I doubt) please report that to https://github.com/DarkShadow44/SereneSeasons/issues and I'll take care of it.

**Comment by iamsoNewBee** [comment]
No crash now. Thanks for ur backport! : )
